### DISCUSSION TITLE
Font used in colorized extruder icons?

### DISCUSSION THREAD

**Discussion by terrapinlogo** [KlipperScreen/KlipperScreen#903] [Q&A]
What font is used? It is listed as "font-family:OpenSans-SemiBold, 'Open Sans'" inside the xml but that doesn't seem to be correct.
Currently working on getting a menu set for the Enraged Rabbit and want to extend the icons up to 12 without it looking jank and changing fonts half way.

**Comment by alfrix** [comment]
i think it was D-Din, but i don't remember exactly, i may re-do them in D-Din and leave a not in the svg
